script1

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Check if at least two parameters are provided
+if test $# -lt 2; then
+    echo "At least 2 parameters are required."
```

Comment:
I think you can remove lines 4-7. The exercise doesn't say to exit early if the user entered less than 2 parameters. In case the user didn't pass any arguments to the script, then you should write: "2nd parameter does not exist"